app/processing/payment/calculate-scheduled-payments/get-schedule.js

```diff
@@ -1,16 +1,17 @@
+const getExpectedSettlementValue = require('./get-expected-settlement-value')
```

Comment:
There are two files named `get-schedule` could consider renaming to more descriptive filenames for ease of understanding. 
`app/processing/components/get-schedule.js`
``app/processing/payment/calculate-scheduled-payments/get-schedule.js

Reply:
Agreed, renamed.